src/global.d.ts

```diff
@@ -0,0 +1 @@
+declare module '*';
```

Comment:
what is for?

Reply:
It used to fix error from typescript of using extend 
```
Could not find a declaration file for module 'extend'. '/Users/fcy/Documents/GitHub/sn-table/node_modules/extend/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/extend` if it exists or add a new declaration (.d.ts) file containing `declare module 'extend';`
```

Reply:
https://www.npmjs.com/package/@types/extend 
`yarn add -D @types/extend` should work, right?

Reply:
Yes, adding @types/extend fixed the issue.